test-utils/runtime/transaction-pool/src/lib.rs

```diff
@@ -97,10 +119,12 @@ impl TestApi {
 	}
 
 	/// Push block under given number.
-	///
-	/// If multiple blocks exists with the same block number, the first inserted block will be
-	/// interpreted as part of the canonical chain.
-	pub fn push_block(&self, block_number: BlockNumber, xts: Vec<Extrinsic>) -> Header {
+	pub fn push_block(
+		&self,
+		block_number: BlockNumber,
+		xts: Vec<Extrinsic>,
+		is_best_block: bool,
```

Comment:
We could probably use `IsBestBlock` enum all the way here, but it's a minor thing.